save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/ProjectView.kt

```diff
@@ -197,19 +181,11 @@ class ProjectView : RComponent<ProjectExecutionRouteProps, ProjectViewState>() {
                 }
                 return
             }
-            state.binaryFile ?: run {
-                setState {
-                    isErrorOpen = true
-                    errorLabel = "No binary file has been selected"
-                    errorMessage = "Please select binary file"
-                }
-                return
-            }
-            state.propertyFile ?: run {
+            if (state.files.isEmpty()) {
```

Comment:
Now we don't check, if user uploaded only one file, for example, just `save.properties`? But we won't be able to work in this case, since bin file is absent

Will it be handled in some other place?

Reply:
I think it becomes really hard to determine which set of files is correct, because for some projects `save.properties` will be optional, for others it would be required; the executable may be a binary or a script (i.e. plaintext) etc. I think it'll be easier to allow arbitrary files for now.

Reply:
Ok, agree